tc/sgx/trusted_worker_manager/enclave/singleton/CMakeLists.txt

```diff
@@ -82,3 +77,6 @@ LINK_ENCLAVE_COMMON_LIBRARIES()
 INCLUDE_ENCLAVE_COMMON_DIRECTORIES()
 INCLUDE_ENCLAVE_COMMON_TARGET_DIRECTORIES()
 TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PRIVATE ${TCF_TOP_DIR}/examples/apps)
+
+# Add $WORKLOAD_STATIC_NAME$ workload
+#SET(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-L,${TCF_TOP_DIR}/examples/apps/build/$WORKLOAD_STATIC_NAME$/workload")
```

Comment:
We need to move all workloads related settings to `CMakeWorkLoads.txt`. 
Please move `CMAKE_SHARED_LINKER_FLAGS` settings related to `CMakeWorkLoads.txt`.
Avalon users who needs to add workloads only need to make changes in one single file

Reply:
Moved all common workload related cmake linker flag settings to a macro inside `CMakeWorkloads.txt`

Reply:
Can you delete above commented lines from this file as it's not relevant

Reply:
Done